ext/vulnsrc/redhat/redhat.go

```diff
@@ -702,11 +702,11 @@ func DbLookupIsAdvisoryProcessed(definition ParsedAdvisory, datastore database.D
 		// error during db lookup, treat advisory as unprocessed
 		return false
 	}
-	if len(foundVulns) > 0 {
-		// found a record, so this has already been processed
-		return true
+	for _, vuln := range foundVulns {
+		if vuln.Valid {
+			return true
```

Comment:
@Allda - based on our conversation in chat - should we only return true if the entire set of fetched vulns matches (as opposed to returning true if any element matches as is the case here), since a given vuln can appear in multiple docs?

Reply:
This was already answered by PST folks internally.